lib/sidekiq/job/iterable/active_record_enumerator.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Sidekiq
+  module Job
+    module Iterable
+      # @private
+      class ActiveRecordEnumerator
```

Comment:
Ideally you'll have a skeleton interface and people can develop their own Enumerators which work with a specific datastore. ActiveRecord is the primary datastore for 90% of Sidekiq apps though; it would make sense to support it well. I'm ok with a single column if it makes the code far simpler.